packages/driver/src/cy/multidomain/index.ts

```diff
@@ -0,0 +1,14 @@
+export function addCommands (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy, state: Cypress.State) {
+  Commands.addAll({
+    switchToDomain (domain, fn) {
+      Cypress.log({
+        name: 'switchToDomain',
+        type: 'parent',
+        message: domain,
+        end: true,
+      })
+
+      Cypress.action('cy:cross:domain:message', 'run:in:domain', fn.toString())
```

Comment:
`fn` here will have the caveat that it doesn't have access to variables of parent lexical scopes, is this something that will be addressed or is it the plan to keep it that way?

Reply:
Yes, it will be addressed in future work. This is a bare-minimum version of this command just to get things rolling.